extensions/wikia/ArticleComments/classes/ArticleComment.class.php

```diff
@@ -632,7 +662,7 @@ public function canEdit() {
 	 * @return bool
 	 */
 	public function isAuthor( $user ) {
-		if ( $this->mUser ) {
+		if ( $this->loadRevisionsAndAuthor( true ) && $this->mUser ) {
```

Comment:
I've asked myself the same question :) It used to be the same in the old implementation and we did not spot any problems. I'd keep it as it is.